docs/source/fx.rst

```diff
@@ -293,10 +293,188 @@ Writing Transformations
 
 TODO
 
-Debugging Transformations
--------------------------
+Debugging
+-----------
+
+Introduction
+^^^^^^^^^^^^^^^^
+
+Once we've generated Python code from our symbolically-traced Module,
+we'll want to verify that the proper semantics were preserved. If they
+weren't, we may need to do some debugging. The key is to work
+backwards: first, check the results of the generated module, then debug
+the generated code, then debug the process of transformations that lead
+to the generated code.
+
+If you’re not familiar with debuggers, please see the auxiliary section
+:ref:`Available Debuggers`.
+
+Debugging the Generated Code
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Because FX generates the ``forward()`` function on GraphModules, using
+traditional debugging techniques like ``print`` statements or ``pdb`` is
+ not as straightfoward. Luckily, we have several techniques we can use
+for debugging the generated code.
+
+``pdb``
+~~~~~~~~~~~~
```

Comment:
The code backticks are making it so that this is not a heading, we can just have "pdb" as a raw stirng